src/script/components/CellsGlobalView/transformCellsNodes/transformCellsNodes.ts

```diff
@@ -44,6 +56,12 @@ export const transformCellsNodes = (nodes: RestNode[]): CellNode[] => {
       ? new Date(Number(node.PreSignedGET?.ExpiresAt) * TIME_IN_MILLIS.SECOND)
       : null;
 
+    const conversationQualifiedId = parseQualifiedId(node.ContextWorkspace?.Uuid || '');
+    const conversation = conversations.find(conversation => conversation.qualifiedId.id === conversationQualifiedId.id);
+
+    const userQualifiedId = getUserQualifiedIdFromNode(node);
+    const user = userQualifiedId ? users.find(user => user.qualifiedId.id === userQualifiedId.id) || null : null;
```

Comment:
Couldn't be more readable? 🤔  Or in return pass user: user || null :)

```suggestion
    const user = users.find(user => user.qualifiedId.id === userQualifiedId?.id) || null;
```